extension/view/js/InPageTranslation.js

```diff
@@ -589,13 +594,29 @@ class InPageTranslation {
 
                         if (!counterpart) {
 
+                            const hasOnlyQEAttributes = node => {
+                                const allowed = new Set([
+                                    "x-bergamot-sentence-index", "x-bergamot-sentence-score",
+                                    "x-bergamot-word-index", "x-bergamot-word-score",
+                                ]);
+
+                                if (node.nodeType === Node.ELEMENT_NODE) {
+                                    if (node.nodeName !== "FONT") return false;
```

Comment:
`.nodeName` is upper-case for HTML, but in XHTML documents it's case-sensitive. You'd either need to compare case-insensitively, or make sure that the generated HTML is all uppercase `<FONT>` instead of `<font>`.